tools/scripts/generic.mk

```diff
@@ -366,7 +367,10 @@ standalone:
 	$(MUTE) $(MAKE) --no-print-directory project LINK_SRCS=n MAKEFLAGS=$(MAKEOVERRIDES)
 
 # Build project using SDK
-project_build: $(PLATFORM)_project_build
+sdkbuild: $(PLATFORM)_project_build
```

Comment:
Let's also rename $(PLATFORM)_project_build ---> $(PLATFORM)_sdkbuild so that it's easier to follow the script. This is internal-only, users wouldn't care, but it makes these scripts more maintainable... same for $(PLATFORM)_project_clean ---> $(PLATFORM)_sdkclean

Reply:
Done